opennms-webapp/src/main/webapp/includes/help-documentation.jsp

```diff
@@ -85,6 +85,18 @@
                             class="btn btn-secondary" role="button" style="width: 100%">Welcome Guide</a></td>
                     <td style="border-top: none;">If you are new in the project, you can find useful information in your Welcome Guide to get anything you need to get started.</td>
                 </tr>
```

Comment:
The Welcome Guide section (lines 85-87) should be removed, as the link doesn't work, and I don't think we have a welcome guide anymore.